### ISSUE TITLE
Enrich the functionality of resharing from RSS feeds to avoid problems and improve publishing

### ISSUE THREAD

**Issue by informapirata** [friendica/friendica#13865]
Good morning everyone.

Disclaimer: **I decided to classify my ISSUE as a feature request and not as a problem, because in fact the problem I have depends above all on the use I made of the function of resharing the RSS feed as if it were one of my Friendica posts. A function that I still find excellent and that I appreciate very much.**

I therefore confirm that I am very satisfied with the Friendica feature that allows me to re-share a website's feed through my account.

I have an account that disseminates information on pirate issues and privacy and therefore I use this resharing system not only with the blogs I own, but also with those sites that deserve to be made known.

However I have a problem.

### Is the feature request related to a problem? Please describe.

The problem is that this service does not give me a way to immediately make it clear that the message published may be the re-sharing of content that is not mine.

### Premise

Before moving on to the request, I want to mention that other software like Mastodon does not offer this functionality natively, although there are services like [mastofeed.org](https://mastofeed.org) (which do not work with Friendica) that allow Mastodon accounts to publish from RSS feeds.

Its functions are certainly more complex to use than Friendica's simple resharing and involve the use of variables `(TITLE)`, `(LINK)`, `(IMAGE)` and `(SUMMARY)` in order to better structure the content of the post.

(TITLE) is the <title> tag.
(LINK) is the permalink in the item. Usually <link> or <guid>
(SUMMARY) contains the first sentences from <description> or <content:encoded>
(IMAGE) will upload the picture for that item. Usually at <image> or <itunes:image> -- alternatively, it will try with the first picture inside the HTML of the description, if there is one.
You can use emojis and line breaks, but no HTML.

```
Example:
(TITLE)
@lemmy_community@lemmy_instance
(SUMMARY)
(LINK)
🏴‍☠ by informapirata 🏴‍☠
@friendica_group@friendica_group
(IMAGE)
```

### Describe the feature you'd like

**I would therefore be happy if your development staff could think of a feature that, in order of importance, is capable of:**
1. **At least allow you to append a disclaimer to the shared feed**
2. **set some simple variables similar to those of the [mastofeed.org](https://mastofeed.org) service**
3. **imagine wonderful solutions as you Friendica developers are used to... :-)**

### Describe alternatives you've considered

- look for a service like [mastofeed.org](https://mastofeed.org) that works with Friendica
- ask the developer of [mastofeed.org](https://mastofeed.org) to make Friendica compatible
- ask you or someone else to create a mastofeed.org for Friendica
- develop it myself, but... oops: this is not possible, because I don't know how to program :-)

### Additional context

Here is a screenshot of the [mastofeed.org](https://mastofeed.org) interface:

![Mastofeed](https://github.com/friendica/friendica/assets/99956935/59e2cf1d-deca-4ad2-a123-87cf40a0f882)

**Comment by annando** [comment]
The system currently supports two ways handling with it. One is that you use the "fetch more information" option on the contact page to not fetch the whole post but to fetch just the link with attached information, just like you would do, when you shared that post by hand.

The other option is that you create a delegated account for each feed that you want to reshare. So you can make sure that people realize that it is not your content. You then can also automatically reshare that new account via your primary account. This means that all your current followers would still receive the same content, but they would realize that isn't your content, but that you reshared it.

**Comment by informapirata** [comment]
> So you can make sure that people realize that it is not your content. You then can also automatically reshare that new account via your primary account. This means that all your current followers would still receive the same content, but they would realize that isn't your content, but that you reshared it.

The multiplication of as many accounts as there are Feeds to be reshared is obviously not a real solution, but rather a patch. Bear in mind that I myself am the administrator of the Friendica poliverso.org instance and if I wanted I could create 144,000 users, one for each feed I wanted, without having to be accountable to anyone but myself. And in fact I have already created four service accounts that re-share Feeds, drawing only and exclusively from some sites selected by topic (Pirates, Privacy, Cybersecurity, etc).

The issue concerns a different issue that impacts the integration of monitored accounts, regarding the publication of spontaneous messages and interactions, and of automatic accounts, regarding the automated publication of some feeds that are particularly important to those who manage that account. accounts.

Please also note that also considering the possibility of republishing on Bluesky, I had to suspend the publication of a particular feed, in which the author spoke in the first person. This unwanted impersonation was particularly noticed by Bluesky users.

In short, it is clear that the problem is not a critical problem (and in fact I have not presented this Issue as a bug), but I simply wanted to explain what the problem is and invite you to think of an intelligent solution, based on the tools available.

**Comment by informapirata** [comment]
> like [mastofeed.org](https://mastofeed.org) (which do not work with Friendica)

A necessary update regarding mastofeed.org

The mastofeed service can finally work with Friendica (when I wrote the issue I was unable to connect my Friendica account with mastofeed).

Mastofeed paradoxically works better with Friendica than with Mastodon, since you can use BBCode to mark up the message, but you must pay attention to the fact that mastofeed is designed to produce messages of up to 500 characters and therefore cannot exploit all the potential of Friendica.

**It's clear: the possibility of using Mastofeed for Friendica makes my issue much less useful; however, I still think that it would be fantastic if we could implement a preventative editing system for Friendica posts that arise from RSS feeds, without having to resort to third-party services.**

**Of course, with the right calm and according to priorities, because I am already very satisfied with how Friendica has improved so much in the last 18 months. For such a long-running project, this is a great result!**

**Comment by kr428** [comment]
Related, I guess: #14081 , #14109 .

**Comment by informapirata** [comment]
> Related, I guess: #14081 , #14109 .

in reality my request is broader because it concerns complete editing in the composition of the feed.

The omission of the title in fact has an ergonomic nature external to Friendica, but most likely it is exclusively aimed at allowing greater readability for mastodon users ([an old problem, this one with the title, currently solved only by Kbin](https://github.com/friendica/friendica/issues/13206)).

On the other hand, I would like the possibility of deciding where to insert the title, where to insert the summary, the possibility of hypertextualizing one of these parts of the text through bbcode formatting, mentioning a user, a Friendica group or a Lemmy community.

In short, what I ask is not linked to the enjoyment of Mastodon users, but precisely to editing reasons

**Comment by kr428** [comment]
Yes, I agree. That was my reference to EchoApp in #14081 about. Title's just one aspect of that.

**Comment by informapirata** [comment]
> Yes I agree. This was my reference to EchoApp in the [# 14081](https://github.com/friendica/friendica/issues/14081) approximately. The title is only one aspect of this.

This issue is also related to https://github.com/friendica/friendica/issues/14194
